<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>741</width>
      <height>521</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="incomeTab">
     <attribute name="title">
      <string>Income</string>
     </attribute>
     <widget class="QTableWidget" name="incomeTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>401</width>
        <height>241</height>
       </rect>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="totalLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Current Income Total:</string>
      </property>
     </widget>
     <widget class="QLabel" name="fulltotalLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>320</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>To Be Determined</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="taxbrakTable">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>50</y>
        <width>291</width>
        <height>411</height>
       </rect>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>20</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Tax Bracket</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Sources of Income</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="expensesTab">
     <attribute name="title">
      <string>Expenses Pie Chart</string>
     </attribute>
     <widget class="QListWidget" name="expensesList">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>30</y>
        <width>251</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
     <widget class="QChartView" name="piechartView" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>421</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Expenses Pie Chart</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>10</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Current Expenses</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>300</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Current Total Expenses:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelExpTotal">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>320</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>To Be Calculated</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuExpenses">
    <property name="title">
     <string>Expenses</string>
    </property>
    <addaction name="actionGroceries_2"/>
   </widget>
   <widget class="QMenu" name="menuIncome">
    <property name="title">
     <string>Income</string>
    </property>
    <addaction name="actionEdit"/>
   </widget>
   <widget class="QMenu" name="menuEmergencies">
    <property name="title">
     <string>Emergencies</string>
    </property>
    <addaction name="actionView"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuExpenses"/>
   <addaction name="menuIncome"/>
   <addaction name="menuEmergencies"/>
  </widget>
  <action name="actionGroceries">
   <property name="text">
    <string>Groceries</string>
   </property>
  </action>
  <action name="actionEdit_Income">
   <property name="text">
    <string>Edit Income</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionGroceries_2">
   <property name="text">
    <string>Groceries</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+G</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="text">
    <string>Edit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+I</string>
   </property>
  </action>
  <action name="actionView">
   <property name="text">
    <string>View and Edit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+E</string>
   </property>
  </action>
  <action name="actionEdit_2">
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header>PyQt5.QtChart</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
